src/Uno.ReferenceImplComparer/Program.cs

```diff
@@ -61,7 +61,7 @@ private static bool CompareAssemblies(AssemblyDefinition referenceAssemby, Assem
 						// and that the hierarchy will be adjusted for wasm to match skia.
 						&& referenceType.BaseType?.FullName != "Windows.UI.Xaml.Shapes.ArbitraryShapeBase")
 					{
-						Console.WriteLine($"{referenceType.FullName} base type is different {referenceType.BaseType?.FullName} in reference, {runtimeType.BaseType?.FullName} in {identifier}");
+						Console.Error.WriteLine($"Error: {referenceType.FullName} base type is different {referenceType.BaseType?.FullName} in reference, {runtimeType.BaseType?.FullName} in {identifier}");
```

Comment:
Can you introduce any error to make sure this works? If it doesn't, then you can try the following:


```suggestion
						Console.Error.WriteLine($"##[error]{referenceType.FullName} base type is different {referenceType.BaseType?.FullName} in reference, {runtimeType.BaseType?.FullName} in {identifier}");
```

https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash

Reply:
Thanks, I tried and even `Console.Error` shows up properly. But thanks for the link, nice to know other types of "output" can be highlighted as well!